webforces/server/core.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, validation=False) -> None:
         logger.debug("Core init")
         self.auth = Auth()
         self.db = MongoDBWorker(validation)
-        self._is_done = True
+        #self._is_done = True
```

Comment:
After the first instantiation of a Core() object, this flag blocks calling self._populateIds() method from connect() method.

Reply:
I think we need to call `_populateIds` manually when it's needed since it's needed not only right after connecting to the database. So we can remove `_populateIds` from connect.

Reply:
Ok.

Reply:
Done.